docs/development/setup-advanced.md

```diff
@@ -379,6 +435,14 @@ Start by [cloning your fork of the zulip repository][zulip-rtd-git-cloning]:
 
 `git clone --config pull.rebase https://github.com/YOURUSERNAME/zulip.git`
 
+As well as [connecting your fork to Zulip upstream with]
+(..git/cloning.html#step-1c-connect-your-fork-to-zulip-upstream)
```

Comment:
For the URL, it'd probably be better to use the `[text][link-name]` with `[link-name]: /git/...` as the choice of Markdown link style to avoid repeating the URL N times; there's several examples of this in the file and throughout our docs.